src/applications/facility-locator/components/markers/FacilityMarker.jsx

```diff
@@ -1,18 +0,0 @@
-import DivMarker from './DivMarker';
```

Comment:
This file is not used or referenced anywhere. It was added to the repo ~4 years ago and the file that referenced it was deleted shortly after.